### PR TITLE
initial outline with Russ Rutledge and Cedric Williams and Noah Cawley

### PR DESCRIPTION
**Pull request by gruetter**
/CC @cewilliams , @rrrutledge, @_utanapishtim

### CODE REVIEW

File: trusted-committer/outline.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    - Product Owner (distinction btw. Guest and Host PO)
+    - TC
+        - TC has rights to push code closer to production than a contributor
+        - also known as _Host_
+        - has privileges that an admin or Apache committer would have but it is
+          a job, irrespective of the permissions. 
+        - Maintainer/Owner (conflict with _maintainer_ role in GitHub)
+        - Committer is not an IS term. Apache uses this term to denote a tech
+          leadership role (TC++)
+        - The TC role is defined by its responsibilities, rather than by its
```
Comment (rrrutledge) [inline trusted-committer/outline.md]: Great quote!

File: trusted-committer/outline.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+          privileges.
+
+- at PayPal, the discussion surrounding the name of the role was very
+  challenging.  i. e.: the name matters.
+
+### Responsibilities
+
+**Tech oriented**
```
Comment (rrrutledge) [inline trusted-committer/outline.md]: Maybe as a counterpoint state that TCs are _not_ responsible for writing guest contributions for them or rewriting contributions that come in not-to-standard.

File: trusted-committer/outline.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+- 
+
+### Lifecycle
+
+- There might well be multiple TCs in one project
+- 
+
+## Ideas
+
+- Possibly two videos: one for being and one for becoming a TC
```
Comment (rrrutledge) [inline trusted-committer/outline.md]: Like this idea.

File: trusted-committer/outline.md (hunk 2 above)
Comment (gruetter) [reply trusted-committer/outline.md]: I agree that they are not responsible for that, indeed. But they should not be passive on that front either. I have always viewed it as part of my job as TC to assist people in making their contributions conform to the required standards. Usually by way of conversations in PRs, sometimes including pseudocode or sketches of algos. I generally avoided rewriting SW for contributors, but it did happen more than once.

File: trusted-committer/outline.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+- The contributor learning path (LP) video should lay out what a contributor
+  can expect from a/his TC
```
Comment (lenucksi) [inline trusted-committer/outline.md]: Sounds like a good idea. #37 should consider this while we're at it.

File: trusted-committer/outline.md (hunk 1 above)
Comment (lenucksi) [reply trusted-committer/outline.md]: 👍

File: trusted-committer/outline.md (hunk 3 above)
Comment (lenucksi) [reply trusted-committer/outline.md]: Actually, that's a nice idea. Did we do any of that (still need to go through all the Screenlight things)? Would we want to have small parts of it included in the contributor segment (#37) along the lines of "from contributor to trusted committer" - that's something that would likely work with open source projects, I'm not sure how well it works for all inner source projects, with the grass-roots inner source projects that should probably be fine.

File: trusted-committer/outline.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+- Idea for a new LP: InnerSource careers
```
Comment (lenucksi) [inline trusted-committer/outline.md]: ...file under "ideas for a second learning path". There surely are quite a number of aspects that could be handled / presented.
I propose to have a open questions / further development / further research session+document after finishing this LP to preserve all the good things for a second / next one.

**Comment by rrrutledge** [comment]
Sounds good to me.

File: trusted-committer/outline.md (hunk 5 above)
Comment (lenucksi) [reply trusted-committer/outline.md]: #62 needs to know about this

Comment (rrrutledge) [reply trusted-committer/outline.md]: Thanks for adding this note.

**Pull request merged**